src/components/SeatMap/SeatMap.js

```diff
@@ -463,6 +464,8 @@ JetsSeatMap.defaultProps = {
     scaleType: DEFAULT_SCALE_TYPE,
     hiddenSeatFeatures: [],
     colorTheme: {
+      seatMapBackgroundColor: THEME_BACKGROUND_COLOR,
```

Comment:
- need to use `seatMapBackgroundColor` as background, not only for wings' leading
 - add documentation